content/en/logs/guide/build-custom-reports-using-log-analytics-api.md

```diff
@@ -0,0 +1,667 @@
+       "to":"1597086120000",
+       "query":"*"
+           },
+   "group_by":[
+       {
+           "type":"facet",
+           "facet":"status",
+           "sort":{
+               "order":"desc"
+           }
+       }
+   ]
+}'
+```
+
+**Response:**
+
+```json
+{
+    "meta": {
+        "status": "done",
+        "request_id": "ZHZlZ1Myek1UMjZDYXZ4am16bFFnUXxIa1BPa3ZwYi1iYW5vM0JzQWNEQ2NB",
+        "elapsed": 429
+    },
+    "data": {
+        "buckets": [
+            {
+                "computes": {
+                    "c0": 2317.284155937053
+                },
+                "by": {
+                    "status": "warn"
+                }
+            },
+            {
+                "computes": {
+                    "c0": 119.5178351086976
+                },
+                "by": {
+                    "status": "ok"
+                }
+            },
+            {
+                "computes": {
+                    "c0": 54.850206927300384
+                },
+                "by": {
+                    "status": "info"
+                }
+            }
+        ]
+    }
+}
+```
+
+Similarly, you can build an `avg` timeseries by setting `type` as `timeseries`.
+
+{{% /tab %}}
+{{% tab "Sum" %}}
+
+With the following API call, you can build a `table` with `sum` of values in a `metric` such as `@scan_d` grouped by the field `service`. The `type` must be `total`.
+
+**API call:**
+
+```bash
+curl -L -X POST 'https://api.datadoghq.com/api/v2/logs/analytics/aggregate' -H 'Content-Type: application/json' -H 'DD-API-KEY: <DATADOG_API_KEY>' -H 'DD-APPLICATION-KEY: <DATADOG_APP_KEY>' --data-raw '{
+   "compute":[
+   {
+       "type":"total",
+       "aggregation":"sum",
+       "metric":"@scan_d"
+   }],
+   "filter": {
+       "from":"1597086000000",
+       "to":"1597086120000",
+       "query":"*"
+           },
+   "group_by":[
+       {
+           "type":"facet",
+           "facet":"service",
+           "sort":{
+               "order":"desc"
+           }
+       }
+   ]
+}'
+```
+Similarly, you can build a `sum` timeseries by setting `type` as `timeseries`.
+
+**Response:**
+
+```json
+{
+    "meta": {
+        "status": "done",
+        "request_id": "SDZMOEZDOW1RUHFaXzc5M1FWSmFTQXxaRHJxZnNuNFVnXzdYRkZ5cjJtMGRB",
+        "elapsed": 412
+    },
+    "data": {
+        "buckets": [
+            {
+                "computes": {
+                    "c0": 30486.0
+                },
+                "by": {
+                    "service": "abc"
+                }
+            },
+            {
+                "computes": {
+                    "c0": 16113.0
+                },
+                "by": {
+                    "service": "xyz"
+                }
+            }
+        ]
+    }
+}
+```
+
+{{% /tab %}}
+{{% tab "Min" %}}
+
+With the following API call, you can build a `table` with `min` of values in a `metric` such as `@scan_d` grouped by the field `service`. The `type` must be `total`.
```

Comment:
Do we really want to highlight values such as `scan_d` instead of `duration` that is used in all integration?

```suggestion
With the following API call, build a `table` with `min` of values in a `metric` such as `@scan_d` grouped by the field `service`. The `type` must be `total`.
```